bitnami/mxnet/templates/scheduler-service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if eq .Values.mode "distributed" }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mxnet.fullname" . }}-scheduler
+  labels:
+    app.kubernetes.io/name: {{ include "mxnet.name" . }}
+    helm.sh/chart: {{ include "mxnet.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "scheduler"
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
I would leave it optional in case you want to have a multi-cluster setup, where you have different workers from different k8s clusters (this makes sense for massive computations)